src/tests/test_cases/sections.py

```diff
@@ -75,7 +90,7 @@ def no_blank_line_after_section():
 
     Returns
     -------
-    A whole lot of values.
+    A value of some sort.
```

Comment:
I split it to a different error code which is not raised by default in the `numpy` convention.